_aicspylibczi/Reader.cpp

```diff
@@ -110,6 +110,10 @@ Reader::dimSizes()
   ans.push_back(sbsize.h);
   ans.push_back(sbsize.w);
 
+  pixelType();
+  if (ImageFactory::numberOfSamples(m_pixelType) == 3)
+    ans.push_back(3);
```

Comment:
Are we sure that it will always be 3? Can it be BGRA?

Reply:
currently libczi doesn't support RGBA but I'll change this, it's sloppy. I should have done > 1 ad then push_back(numberOfSamples)